tools/container-converter/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde = { version = "1.0.127", features = ["derive"] }
 serde_json =  { git = "https://github.com/fortanix/serde-json.git", branch = "base64_bytes" }
 shiplift = { git = "https://github.com/fortanix/shiplift.git" }
 sys-mount = "3"
-tar = "0.4.39"
+tar = { git = "https://github.com/alexcrichton/tar-rs" }
```

Comment:
Why is this needed? Patching a crate from crates.io should be by using a patch section.

Reply:
Oh yeah, patch is correct here, I forgot about the patch. I need to revert this because it crashes `tar` reading in container converter.